rosa_learning/creating_cluster_workshop/cloud-experts-getting-started-managing-worker-nodes.adoc

```diff
@@ -0,0 +1,233 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-getting-started-managing-worker-nodes"]
+= Tutorial: Managing worker nodes
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-getting-started-managing-worker-nodes
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 2023-11-30
+
+In {product-title} (ROSA), changing aspects of your worker nodes is performed through the use of machine pools. A machine pool allows users to manage many machines as a single entity. Every ROSA cluster has a default machine pool that is created when the cluster is created. For more information, see the xref:../../rosa_cluster_admin/rosa_nodes/rosa-nodes-machinepools-about.adoc#rosa-nodes-machinepools-about[machine pool] documentation.
+
+
+== Creating a machine pool
+You can create a machine pool with either the command line interface (CLI) or the user interface (UI). 
+
+=== Creating a machine pool with the CLI
+. Run the following command:
++
+[source,terminal]
+----
+rosa create machinepool --cluster=<cluster-name> --name=<machinepool-name> --replicas=<number-nodes>
+----
++
+.Example input
++
+[source,terminal]
+----
+ $ rosa create machinepool --cluster=my-rosa-cluster --name=new-mp
+ --replicas=2
+----
++
+.Example output
++
+[source,terminal]
+----
+I: Machine pool 'new-mp' created successfully on cluster 'my-rosa-cluster'
+I: To view all machine pools, run 'rosa list machinepools -c my-rosa-cluster'
+----
+
+. *Optional:* Add node labels or taints to specific nodes in a new machine pool by running the following command:
++
+[source,terminal]
+---- 
+rosa create machinepool --cluster=<cluster-name> --name=<machinepool-name> --replicas=<number-nodes> --labels=`<key=pair>`
+----
++
+.Example input
++
+[source,terminal]
+----
+$ rosa create machinepool --cluster=my-rosa-cluster --name=db-nodes-mp --replicas=2 --labels='app=db','tier=backend'
+----
++
+.Example output
++
+[source,terminal]
+----
+I: Machine pool 'db-nodes-mp' created successfully on cluster 'my-rosa-cluster'
+----
++
+This creates an additional 2 nodes that can be managed as a unit and also assigns them the labels shown.  
+
+. Run the following command to confirm machine pool creation and the assigned labels:
++
+[source,terminal]
+----
+rosa list machinepools --cluster=<cluster-name>
+----
++
+.Example output
++
+[source,terminal]
+----
+ID          AUTOSCALING  REPLICAS  INSTANCE TYPE  LABELS            TAINTS    AVAILABILITY ZONES
+Default     No           2         m5.xlarge                                  us-east-1a
+----
+
+=== Creating a machine pool with the UI
+. Log in to the {cluster-manager-url} and click your cluster.
++
+image::cloud-experts-getting-started-managing-ocm-cluster.png[]
+
+. Click *Machine pools*.
++
+image:cloud-experts-getting-started-managing-mp-ocm.png[]
+
+. Click *Add machine pool*.
+
+. Enter the desired configuration.
```

Comment:
User needs to click "Add machine pool" button to add a new machine pool at the end. Please add the same.